.gitignore

```diff
@@ -1,3 +1,2 @@
 /node_modules
-/lib
```

Comment:
Why did you do this? This folder contains build artifacts and should be created via npm hooks etc.